infra/terraform/modules/data_access/organization_events/organization_events.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+
+import boto3
+
+
+LAMBDA_ARNS = {
+    "create_user_role": os.environ["CREATE_ROLE_ARN"],
+    "delete_user_role": os.environ["DELETE_ROLE_ARN"],
+}
+
+
+def event_received(sns_event, context):
+    print("SNS event received = {}".format(json.dumps(sns_event)))
+    for record in sns_event["Records"]:
+        event = json.loads(record["Sns"]["Message"])
+        action = event["action"]
+        if action == "member_added":
+            member_added(event)
+        elif action == "member_removed":
+            member_removed(event)
+
+
+def member_added(event):
+    print("Handling 'member_added' event: {}".format(event))
+
+    invoke_lambda(
+        function=LAMBDA_ARNS["create_user_role"],
+        payload=payload(event)
+    )
+
+
+def member_removed(event):
+    print("Handling 'member_removed' event: {}".format(event))
+
+    invoke_lambda(
+        function=LAMBDA_ARNS["delete_user_role"],
+        payload=payload(event)
+    )
+
+
+def invoke_lambda(function, payload):
+    client = boto3.client("lambda")
+    client.invoke(
+        FunctionName=function,
+        Payload=bytes(json.dumps(payload), "utf8"),
```

Comment:
Not familiar with Python sorry, I'll give it a go.

I didn't know strings have this method, would the returned value compatibles with Boto's Payload param (bytes or seekable file-like object)?

PS: It ASCII encoding is already UTF8 as ASCII is a subset of UTF8 which is retro-compatible with it.
UTF8 is a variable length encoding but the encoding for the most common chars, e.g. ASCII chars, use less bytes (1 in the case of ASCII) and less common chars use 2-4 bytes.